apps/journeys-admin/src/components/Editor/EditToolbar/EditToolbar.tsx

```diff
@@ -28,6 +29,7 @@ export function EditToolbar(): ReactElement {
     >
       {journey != null && (
         <>
+          {journey != null && <Analytics journey={journey} variant="button" />}
```

Comment:
This check is already been done
```suggestion
          {<Analytics journey={journey} variant="button" />}
```